k8s/charts/openebs/values.yaml

```diff
@@ -128,29 +124,33 @@ webhook:
 
 jiva:
   image: "quay.io/openebs/jiva"
-  imageTag: "1.4.0"
+  imageTag: "1.5.0-RC2"
   replicas: 3
   defaultStoragePath: "/var/openebs"
 
 cstor:
   pool:
     image: "quay.io/openebs/cstor-pool"
-    imageTag: "1.4.0"
+    imageTag: "1.5.0-RC2"
   poolMgmt:
     image: "quay.io/openebs/cstor-pool-mgmt"
-    imageTag: "1.4.0"
+    imageTag: "1.5.0-RC2"
   target:
     image: "quay.io/openebs/cstor-istgt"
-    imageTag: "1.4.0"
+    imageTag: "1.5.0-RC2"
   volumeMgmt:
     image: "quay.io/openebs/cstor-volume-mgmt"
-    imageTag: "1.4.0"
+    imageTag: "1.5.0-RC2"
+
+helper:
+  image: "quay.io/openebs/linux-utils"
+  imageTag: "1.5.0"
```

Comment:
Add this to the README. Can you also modify the openebs-operator to take the HELPER image env on maya-apiserver.